README.md

```diff
@@ -217,7 +217,10 @@ Make sure version 1.1 or later of the DSC Linux agent is installed on the machin
 **To configure Azure Automation as a DSC Pull Server from the Linux computer:**
 
 - On each Linux machine to onboard to Azure Automation DSC, use Register.py to onboard using the PowerShell DSC Local Configuration Manager defaults:  
+#### For python2
 ```/opt/microsoft/dsc/Scripts/Register.py <Automation account registration key> <Automation account registration URL>  ```
```

Comment:
There are few other references to the script folder in the same readme. Please consider updating them too.

Reply:
All the references to the /Scripts folder have been updated to show both python2 and python3 paths.